src/guide/advanced/triggers.md

```diff
@@ -242,7 +266,7 @@ much to worry about.
 
 :::
 
-### Pre-commit Triggers
+#### Pre-commit Triggers
 
 This is a variant of timed triggers that gets run before transactions get
 committed. It leaves a special event to be triggered in the next block.
```

Comment:
Technicaly *Pre-commit Triggers* are executed just before **block** commit. And block can contain one ore more transactions